orchagent/bufferorch.cpp

```diff
@@ -80,24 +101,49 @@ void BufferOrch::initBufferReadyList(Table& table)
     // populate the lists with buffer configuration information
     for (const auto& key: keys)
     {
-        m_ready_list[key] = false;
-
         auto tokens = tokenize(key, config_db_key_delimiter);
         if (tokens.size() != 2)
         {
             SWSS_LOG_ERROR("Wrong format of a table '%s' key '%s'. Skip it", table.getTableName().c_str(), key.c_str());
             continue;
         }
 
+        // We need transform the key from config db format to appl db format
+        auto appldb_key = tokens[0] + delimiter + tokens[1];
```

Comment:
account for the table name change

Reply:
Actually, we should not change this because this consists of keys only without the table name part.